addons/l10n_ar_withholding/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Argentina - Payment Withholdings',
-    'version': "1.0",
+    'version': "1.1",
```

Comment:
For changes that don't require a module upgrade script, there's no need to bump the module version.

```suggestion
    'version': "1.0",
```